src/sidebar/templates/thread-list.html

```diff
@@ -1,19 +1,23 @@
 <ul class="thread-list">
   <li class="thread-list__spacer"
       ng-style="{height: vm.virtualThreadList.offscreenUpperHeight}"></li>
-  <li id="{{child.id}}"
-      class="thread-list__card"
-      ng-mouseenter="vm.onFocus({annotation: child.annotation})"
-      ng-click="vm.onSelect({annotation: child.annotation})"
-      ng-mouseleave="vm.onFocus({annotation: null})"
-      ng-repeat="child in vm.virtualThreadList.visibleThreads track by child.id">
-      <annotation-thread
-        thread="child"
-        show-document-info="vm.showDocumentInfo"
-        on-change-collapsed="vm.onChangeCollapsed({id: id, collapsed: collapsed})"
-        on-force-visible="vm.onForceVisible({thread: thread})">
-      </annotation-thread>
-  </li>
+  <div ng-repeat="child in vm.virtualThreadList.visibleThreads track by child.id">
```

Comment:
I'm not sure off the top of my head what the consequences are of having a DOM structure like this:

```
ul
|- div
  |- li
|- div
  |- li
```

Instead of

```
ul
|- li
    |- child
|- li
    |- child
```

But conceptually it looks wrong and I worry we might be breaking something. Could you investigate?